dk.gqrx.gqrx.yaml

```diff
@@ -0,0 +1,311 @@
+    sources:
+      - type: git
+        url: https://github.com/gnuradio/pmt.git
+        # there are no tags
+        commit: 5960738cd35827acceb5a4de9e1c4005d2d02305
+      - type: patch
+        path: patches/pmt-refl-cpp.patch
+
+  - name: soapy
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/pothosware/SoapySDR.git
+        tag: soapy-sdr-0.8.1
+        commit: 1cf5a539a21414ff509ff7d0eedfc5fa8edb90c6
+
+  - name: soapy_remote
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/pothosware/SoapyRemote.git
+        tag: soapy-remote-0.5.2
+        commit: f920d9bf10f62f67c8e31b7dc25090bc784e5210
+
+  - name: soapy_uhd
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/pothosware/SoapyUHD.git
+        # latest master for multiple fixes
+        commit: 6b521393cc45c66770f3d4bc69eac7dda982174c
+
+  - name: portaudio
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/PortAudio/portaudio.git
+        tag: v19.7.0
+        commit: 147dd722548358763a8b649b3e4b41dfffbcfbb6
+
+  - name: gnuradio
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DENABLE_DEFAULT=OFF
+      - -DENABLE_EXAMPLES=OFF
+      - -DENABLE_TESTING=OFF
+      - -DENABLE_PYTHON=OFF
+      - -DENABLE_POSTINSTALL=ON
+      - -DENABLE_DOXYGEN=OFF
+      - -DENABLE_MANPAGES=OFF
+      - -DENABLE_GNURADIO_RUNTIME=ON
+      - -DENABLE_COMMON_PCH=OFF
+      - -DENABLE_GR_CTRLPORT=OFF
+      - -DENABLE_GRC=OFF
+      - -DENABLE_GR_BLOCKS=ON
+      - -DENABLE_GR_FEC=OFF
+      - -DENABLE_GR_FFT=ON
+      - -DENABLE_GR_FILTER=ON
+      - -DENABLE_GR_ANALOG=ON
+      - -DENABLE_GR_DIGITAL=ON
+      - -DENABLE_GR_DTV=OFF
+      - -DENABLE_GR_AUDIO=ON
+      - -DENABLE_GR_CHANNELS=OFF
+      - -DENABLE_GR_PDU=OFF
+      - -DENABLE_GR_IIO=OFF
+      - -DENABLE_GR_QTGUI=OFF
+      - -DENABLE_GR_TRELLIS=OFF
+      - -DENABLE_GR_UHD=ON
+      - -DENABLE_UHD_RFNOC=ON
+      - -DENABLE_GR_UTILS=OFF
+      - -DENABLE_GR_MODTOOL=OFF
+      - -DENABLE_GR_BLOCKTOOL=OFF
+      - -DENABLE_GR_VIDEO_SDL=OFF
+      - -DENABLE_GR_VOCODER=OFF
+      - -DENABLE_GR_WAVELET=OFF
+      - -DENABLE_GR_ZEROMQ=OFF
+      - -DENABLE_GR_NETWORK=ON
+      - -DENABLE_GR_SOAPY=ON
+    builddir: true
+    sources:
+      - type: git
+        url: https://github.com/gnuradio/gnuradio.git
+        tag: v3.10.12.0
+        commit: ec7a3239645162ffea1925e6f1e6e62bf1349d03
+
+  - name: gr-osmosdr
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://gitea.osmocom.org/sdr/gr-osmosdr.git
+        tag: v0.2.6
+        commit: a6afeaa3a1038e64751616d769537cd22ace015d
+
+  - name: gqrx
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/gqrx-sdr/gqrx.git
+        tag: v2.17.7
+        commit: 1a8ab3a3cc02db3bf4c9058ed4b60ddad06fe9d1
+        x-checker-data:
```

Comment:
x-checker-data enabled only for the Gqrx module for now since the other ones in many cases bring breaking changes and should be updated manually. I may enable x-checker-data for some of them later.